selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/RenameOrganizationTest.java

```diff
@@ -74,81 +72,69 @@ public void setUp() throws Exception {
 
   @AfterClass
   public void tearDown() throws Exception {
-    testOrganizationServiceClient.deleteById(parentOrganization.getId());
+    for (OrganizationDto organization : testOrganizationServiceClient.getAll())
+      testOrganizationServiceClient.deleteById(organization.getId());
   }
 
   @Test(priority = 1)
   public void testParentOrganizationRename() {
     navigationBar.waitNavigationBar();
-    navigationBar.clickOnMenu(NavigationBar.MenuItem.ORGANIZATIONS);
+    navigationBar.clickOnMenu(ORGANIZATIONS);
     organizationListPage.waitForOrganizationsToolbar();
     organizationListPage.waitForOrganizationsList();
 
+    // Open the parent organization and try to set is not valid name
     organizationListPage.clickOnOrganization(parentOrganization.getName());
-
     organizationPage.waitOrganizationTitle(parentOrganization.getName());
-
-    organizationPage.setOrganizationName("");
+    organizationPage.setOrganizationName(" ");
     editMode.waitDisplayed();
     assertFalse(editMode.isSaveEnabled());
-
     editMode.clickCancel();
     editMode.waitHidden();
     assertEquals(parentOrganization.getName(), organizationPage.getOrganizationName());
 
-    organizationPage.setOrganizationName(parentNewName);
+    // Test renaming of the parent organization
+    organizationPage.setOrganizationName(NEW_PARENT_ORG_NAME);
     editMode.waitDisplayed();
     assertTrue(editMode.isSaveEnabled());
-
     editMode.clickSave();
     editMode.waitHidden();
-
-    organizationPage.waitOrganizationTitle(parentNewName);
-    assertEquals(parentNewName, organizationPage.getOrganizationName());
+    organizationPage.waitOrganizationTitle(NEW_PARENT_ORG_NAME);
+    assertEquals(NEW_PARENT_ORG_NAME, organizationPage.getOrganizationName());
   }
 
   @Test(priority = 2)
   public void testSubOrganizationRename() {
+    String organizationPath = NEW_PARENT_ORG_NAME + "/" + CHILD_ORG_NAME;
+    String path = NEW_PARENT_ORG_NAME + "/" + NEW_CHILD_ORG_NAME;
```

Comment:
To avoid confusion, it's better to rename **path** variable to describe its propose explicitly compere to **organizationPath** variable.